### PR TITLE
removed unecessary view files and including views from looker-hub instead

### PR DESCRIPTION
**Pull request by kik-kik**
# removed unnecessary view files and including views from looker-hub instead

Realized there is unnecessary complexity with the files in this folder.

### CODE REVIEW

File: websites/websites.model.lkml [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
+include: "//looker-hub/websites/views/*.view.lkml"
 
-explore: blogs_summary{}
-explore: landing_page_metrics{}
-explore: metrics_summary{}
-explore: page_metrics{}
+explore: blogs_daily_summary{}
+explore: moz_org_landing_page_metrics{}
+explore: moz_org_metrics_summary{}
+explore: moz_org_page_metrics{}
```
Comment (ANich) [inline websites/websites.model.lkml:9]: Are there any dashboards/looks currently built with these? They might break due to the name-change

File: websites/views/blogs_summary.view.lkml [hunk 2]
```diff
@@ -1,5 +0,0 @@
-include: "//looker-hub/websites/views/blogs_daily_summary.view.lkml"
-
-view: blogs_summary {
-  extends: [blogs_daily_summary]
-}
```
Comment (ANich) [inline websites/views/blogs_summary.view.lkml:5]: Does this data not generally have any metrics?

**Review by ANich — APPROVED** [review]
LGTM - left a few questions

File: websites/websites.model.lkml (hunk 1 above)
Comment (kik-kik) [reply websites/websites.model.lkml]: I don't believe there are.

File: websites/views/blogs_summary.view.lkml (hunk 2 above)
Comment (kik-kik) [reply websites/views/blogs_summary.view.lkml:5]: Added some measures @ANich, would be amazing if you could re-review the PR.

**Review by fbertsch — APPROVED** [review]
Looks fine, you could consider adding hidden explores with the old names to ensure backwards compatibility.

File: websites/views/moz_org_metrics_summary.view.lkml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  measure: downloads {
+    label: "Download Count"
+    type: sum
+    sql: ${TABLE}.downloads;;
+  }
+
+  measure:  non_fx_downloads {
+    label: "Non-Fx Download Count"
+    type: sum
+    sql: ${TABLE}. non_fx_downloads;;
```
Comment (ANich) [inline websites/views/moz_org_metrics_summary.view.lkml:25]:
```suggestion
    sql: ${TABLE}.non_fx_downloads;;
```

File: websites/views/moz_org_landing_page_metrics.view.lkml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  measure: downloads {
+    label: "Download Count"
+    type: sum
+    sql: ${TABLE}.downloads;;
+  }
+
+  measure:  non_fx_downloads {
+    label: "Non-Fx Download Count"
+    type: sum
+    sql: ${TABLE}. non_fx_downloads;;
```
Comment (ANich) [inline websites/views/moz_org_landing_page_metrics.view.lkml:25]:
```suggestion
    sql: ${TABLE}.non_fx_downloads;;
```

File: websites/websites.model.lkml [hunk 5]
```diff
@@ -1,9 +1,10 @@
 connection: "telemetry"
 label: "Websites"
 
-include: "/websites/views/*.view.lkml"
+include: "./views/*.view.lkml"
```
Comment (ANich) [inline websites/websites.model.lkml:4]: IIRC Looker does relative imports by default. Feel free to ignore
```suggestion
include: "views/*"
```

**Review by ANich — COMMENTED** [review]
A few typos and one recommendation

**Pull request merged**